features/media-import.feature

```diff
@@ -69,6 +69,24 @@ Feature: Manage WordPress attachments
       My fabulous caption
       """
 
+  Scenario: Import a file as attachment from a local image and leave it in it's current location
+    Given download:
+      | path                        | url                                              |
+      | {CACHE_DIR}/large-image.jpg | http://wp-cli.org/behat-data/large-image.jpg     |
+
+    When I run `wp media import {CACHE_DIR}/large-image.jpg --import_only`
+    Then STDOUT should contain:
+      """
+      Imported file
+      """
+    And STDOUT should contain:
+      """
+      Success: Imported 1 of 1 items.
+      """
+    And the {CACHE_DIR}/large-image.jpg file should exist
```

Comment:
@danielbachhuber I'm struggling to work out how to write this.
Can you point me ot any docs or examples that would help? 

I guess I need to learn what Behat actually does with the image if it imports it? Does it move to `{CACHE_DIR}/wp-content/uploads/year/month`?
Then I *think* I'll need to create a new variable in FeatureContext.php for {UPLOADS_DIR} to use in the test. Is that about right?

Thanks!